components/layout/HamburgerMenu/HamburgerMenu.tsx

```diff
@@ -110,26 +102,8 @@ const HamburgerMenu = (props: HamburgerMenuProps) => {
               </Box>
             </CategoryNestedNavigation>
           </Box>
-          <Box sx={{ ...styles.spacer }}></Box>
-          <List sx={{ ...styles.navLinksList }}>
```

Comment:
We need this list of navigation items to remain the same

Reply:
Added above list in hamburger menu